src/Uno.Extensions.Reactive.UI.Tests/Presentation/Bindings/Collections/Given_BindableCollection_Selection.cs

```diff
@@ -107,4 +181,17 @@ private async Task<(BindableGiven_BindableCollection_Selection_Model, ListView,
 
 		return (vm, lv, lvItems);
 	}
+
+	private async Task<(BindableGiven_BindableCollection_Selection_Model, ComboBox, ICollectionView, MyItem[])> SetupComboBox()
+	{
+		var vm = new BindableGiven_BindableCollection_Selection_Model();
+		var itemsView = new CollectionViewSource { Source = vm.Items }.View;
+		var cb = new ComboBox { ItemsSource = itemsView };
```

Comment:
We don't need (nor want) a `CollectionViewSource`:

```suggestion
		var cb = new ComboBox { ItemsSource = vm.Items };
```